test/server/cards/09.2-BoB/UtakuRumaru.spec.js

```diff
@@ -0,0 +1,112 @@
+                this.game.checkGameState(true);
+                expect(this.rumaru.glory).toBe(2);
+            });
+
+            it('should not effect opponent characters glory', function() {
+                this.outrider.honor();
+                this.game.checkGameState(true);
+                expect(this.outrider.glory).toBe(2);
+            });
+
+            it('should trigger after she wins a conflict', function() {
+                this.noMoreActions();
+                this.initiateConflict({
+                    attackers: [this.rumaru, this.adept],
+                    defenders: [this.outrider]
+                });
+                this.noMoreActions();
+                expect(this.player1).toBeAbleToSelect(this.rumaru);
+            });
+
+            it('should allow you to honor another pariticipating character', function() {
+                this.noMoreActions();
+                this.initiateConflict({
+                    attackers: [this.rumaru, this.adept],
+                    defenders: [this.outrider]
+                });
+                this.noMoreActions();
+                this.player1.clickCard(this.rumaru);
+                expect(this.player1).toBeAbleToSelect(this.adept);
+            });
+
+            it('should honor the chosen character', function() {
+                this.noMoreActions();
+                this.initiateConflict({
+                    attackers: [this.rumaru, this.adept],
+                    defenders: [this.outrider]
+                });
+                this.noMoreActions();
+                this.player1.clickCard(this.rumaru);
+                this.player1.clickCard(this.adept);
+                expect(this.player1).toHavePrompt('Select card to discard');
+                this.player1.clickCard(this.katana);
+                expect(this.adept.isHonored).toBe(true);
+                expect(this.katana.location).toBe('conflict discard pile');
+            });
+
+            it('should not let her honor herself', function() {
+                this.noMoreActions();
+                this.initiateConflict({
+                    attackers: [this.rumaru, this.adept],
+                    defenders: [this.outrider]
+                });
+                this.noMoreActions();
+                this.player1.clickCard(this.rumaru);
+                expect(this.player1).not.toBeAbleToSelect(this.rumaru);
+            });
+
+            it('should not let you honor characters at home', function() {
+                this.noMoreActions();
+                this.initiateConflict({
+                    attackers: [this.rumaru, this.adept],
+                    defenders: [this.outrider]
+                });
+                this.noMoreActions();
+                this.player1.clickCard(this.rumaru);
+                expect(this.player1).not.toBeAbleToSelect(this.scholar);
+            });
+
+            it('should not let you honor your opponent characters', function() {
```

Comment:
At least the spoiler I've seen, it *should* allow you to honor your opponent's characters